src/session.ts

```diff
@@ -755,26 +764,22 @@ export class SessionManager implements Middleware {
                 new SessionMenuItem(
                     "Restart Current Session",
                     () => { this.restartSession(); }),
+
+                ...powerShellItems,
             ];
+
         } else if (this.sessionStatus === SessionStatus.Failed) {
             menuItems = [
                 new SessionMenuItem(
                     `Session initialization failed, click here to show PowerShell extension logs`,
                     () => { vscode.commands.executeCommand("PowerShell.ShowLogs"); }),
+
+                new SessionMenuItem(
```

Comment:
This is in the above block too. Maybe it should just be there in all cases (outside the `if`/`else`)?

Reply:
big refactor since the last commit. This is addressed now.